cmds/ps/ps_linux.go

```diff
@@ -194,24 +194,29 @@ func (p process) getTime() string {
 	jiffies := utime + stime
 
 	tsecs := jiffies / USER_HZ
-	secs := int(tsecs % 60)
-	mins := int((tsecs / 60) % 60)
+	secs := tsecs % 60
+	mins := (tsecs / 60) % 60
 	hrs := tsecs / 3600
 
 	return fmt.Sprintf("%02d:%02d:%02d", hrs, mins, secs)
 }
 
 // Create a ProcessTable containing stats on all processes.
 func (pT *ProcessTable) LoadTable() error {
-	// Match all files and directories directly inside of /proc.
-	matches, err := filepath.Glob(filepath.Join(proc, "*"))
+	// Open and Readdir /proc.
```

Comment:
I'm good with this but one could  just as easily make the glob be [0-9]*? Anyway it's fine.